extensions/positron-run-app/src/positron-run-app.d.ts

```diff
@@ -36,14 +36,12 @@ export interface RunAppOptions {
 	 *
 	 * @param runtime The language runtime metadata for the document's language.
 	 * @param document The document to run.
-	 * @param port The port to run the application on, if known.
```

Comment:
We'll also need to update the copied version of this file in the `positron-python` extension. I usually just copy it over and then run `yarn format-fix` from `positron-python`.